modules/exploits/linux/local/bash_profile_persistence.rb

```diff
@@ -39,6 +39,11 @@ def initialize(info = {})
           'Compat'       =>
           {
             'PayloadType' => 'cmd'
```

Comment:
We're missing a `,` here 👀

Reply:
Got that fixed. Cheers 👍

Reply:
did you fix it manually or was it a problem with the rubocop rules you could fix?

Reply:
I fixed that one manually, but I'm looking into getting it added into the cop at the moment

Reply:
Got a fix pushed up to the [rubocop](https://github.com/rapid7/metasploit-framework/pull/15295) rule to handle that scenario 👍